<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsignaturaDialog</class>
 <widget class="QDialog" name="AsignaturaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asignatura</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>

  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin"><number>12</number></property>
   <property name="topMargin"><number>12</number></property>
   <property name="rightMargin"><number>12</number></property>
   <property name="bottomMargin"><number>12</number></property>
   <property name="spacing"><number>10</number></property>

   <!-- TÍTULO -->
   <item>
    <widget class="QLabel" name="lblTitulo">
     <property name="font">
      <font>
       <family>Dubai</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Asignatura</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>

   <!-- FORMULARIO -->
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="horizontalSpacing"><number>10</number></property>
     <property name="verticalSpacing"><number>8</number></property>

     <item row="0" column="0">
      <widget class="QLabel" name="lblNombre">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txt_nombre"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="lblCreditos">
       <property name="text">
        <string>Créditos:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sp_creditos">
       <property name="minimum"><number>1</number></property>
       <property name="maximum"><number>30</number></property>
      </widget>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="lblCurso">
       <property name="text">
        <string>Curso:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cb_curso">
       <item><property name="text"><string>1</string></property></item>
       <item><property name="text"><string>2</string></property></item>
       <item><property name="text"><string>3</string></property></item>
       <item><property name="text"><string>4</string></property></item>
      </widget>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="lblCuatrimestre">
       <property name="text">
        <string>Cuatrimestre:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cb_cuatrimestre">
       <item><property name="text"><string>1</string></property></item>
       <item><property name="text"><string>2</string></property></item>
      </widget>
     </item>

     <item row="4" column="0">
      <widget class="QLabel" name="lblGrado">
       <property name="text">
        <string>Grado:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cb_grado"/>
     </item>

     <item row="5" column="0">
      <widget class="QLabel" name="lblObligatoria">
       <property name="text">
        <string>Obligatoria:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="chk_obligatoria">
       <property name="text">
        <string>Sí</string>
       </property>
      </widget>
     </item>

    </layout>
   </item>

   <!-- BOTONES -->
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <property name="spacing"><number>10</number></property>
     <item>
      <spacer name="spacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_guardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
